widget/tabcontainer.go

```diff
@@ -80,6 +81,8 @@ func (t *TabContainer) CurrentTab() *TabItem {
 func (t *TabContainer) SelectTabIndex(index int) {
 	if index < 0 || index >= len(t.Items) {
 		return
+	} else if t.current == index {
```

Comment:
This could be merged into the if above:
```
if index < 0 || index >= len(t.Items) || t.current == index {
  return
}
```

Reply:
I thought it would be more readable this way, but probably you are right.

Reply:
Its a tough call, if the bounds checking returned an error then it would be better to keep them separate because you're adding an optimization, but since the outcome is the same we might as well aim for brevity.